<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>2 Exploraci√≥n interactiva de datos transcript√≥micos con iSEE | Mini curso mayo 2021: plotly e iSEE</title>
  <meta name="description" content="2 Exploraci√≥n interactiva de datos transcript√≥micos con iSEE | Mini curso mayo 2021: plotly e iSEE" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="2 Exploraci√≥n interactiva de datos transcript√≥micos con iSEE | Mini curso mayo 2021: plotly e iSEE" />
  <meta property="og:type" content="book" />
  
  
  
  <meta name="github-repo" content="ComunidadBioInfo/minicurso_mayo_2021" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="2 Exploraci√≥n interactiva de datos transcript√≥micos con iSEE | Mini curso mayo 2021: plotly e iSEE" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="icon_32.png" type="image/x-icon" />
<link rel="prev" href="exploraci√≥n-interactiva-de-datos-con-plotly.html"/>

<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-3.5.1/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.3/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.57.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>General</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#c√≥digo-de-conducta"><i class="fa fa-check"></i>C√≥digo de Conducta</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#datos-generales"><i class="fa fa-check"></i>Datos Generales</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#descripci√≥n"><i class="fa fa-check"></i>Descripci√≥n</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#pre-requisitos"><i class="fa fa-check"></i>Pre-requisitos</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#para-qui√©n-va-dirigido-este-mini-curso"><i class="fa fa-check"></i>Para qui√©n va dirigido este mini curso</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#actividades"><i class="fa fa-check"></i>Actividades</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#organizadores"><i class="fa fa-check"></i>Organizadores</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#formulario-de-registro"><i class="fa fa-check"></i>Formulario de registro</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#informaci√≥n-sesi√≥n-de-r"><i class="fa fa-check"></i>Informaci√≥n sesi√≥n de R</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#licencia"><i class="fa fa-check"></i>Licencia</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="exploraci√≥n-interactiva-de-datos-con-plotly.html"><a href="exploraci√≥n-interactiva-de-datos-con-plotly.html"><i class="fa fa-check"></i><b>1</b> Exploraci√≥n interactiva de datos con plotly</a>
<ul>
<li class="chapter" data-level="1.1" data-path="exploraci√≥n-interactiva-de-datos-con-plotly.html"><a href="exploraci√≥n-interactiva-de-datos-con-plotly.html#introducci√≥n-a-plotly"><i class="fa fa-check"></i><b>1.1</b> Introducci√≥n a plotly</a></li>
<li class="chapter" data-level="1.2" data-path="exploraci√≥n-interactiva-de-datos-con-plotly.html"><a href="exploraci√≥n-interactiva-de-datos-con-plotly.html#graficando-datos-de-palmerpenguins-con-plotly"><i class="fa fa-check"></i><b>1.2</b> Graficando datos de palmerpenguins con plotly</a></li>
<li class="chapter" data-level="1.3" data-path="exploraci√≥n-interactiva-de-datos-con-plotly.html"><a href="exploraci√≥n-interactiva-de-datos-con-plotly.html#graficando-datos-de-covid-con-plotly"><i class="fa fa-check"></i><b>1.3</b> Graficando datos de COVID con plotly</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html"><i class="fa fa-check"></i><b>2</b> Exploraci√≥n interactiva de datos transcript√≥micos con iSEE</a>
<ul>
<li class="chapter" data-level="2.1" data-path="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#infraestructura-para-datos-tr√°nscriptomicos"><i class="fa fa-check"></i><b>2.1</b> Infraestructura para datos tr√°nscriptomicos</a></li>
<li class="chapter" data-level="2.2" data-path="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#datos-p√∫blicos"><i class="fa fa-check"></i><b>2.2</b> Datos p√∫blicos</a></li>
<li class="chapter" data-level="2.3" data-path="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#explorando-con-isee"><i class="fa fa-check"></i><b>2.3</b> Explorando con iSEE</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#sce"><i class="fa fa-check"></i><b>2.3.1</b> sce</a></li>
<li class="chapter" data-level="2.3.2" data-path="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#ejercicio"><i class="fa fa-check"></i><b>2.3.2</b> Ejercicio</a></li>
<li class="chapter" data-level="2.3.3" data-path="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#datos-reales"><i class="fa fa-check"></i><b>2.3.3</b> Datos reales</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#m√°s-material"><i class="fa fa-check"></i><b>2.4</b> M√°s material</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Mini curso mayo 2021: plotly e iSEE</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Exploraci√≥n interactiva de datos transcript√≥micos con iSEE</h1>
<p><em>El c√≥digo de R de este archivo est√° disponbie v√≠a <a href="https://github.com/ComunidadBioInfo/minicurso_mayo_2021/blob/main/02_iSEE.R">GitHub</a>.</em></p>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb32-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Carguemos las librer√≠as que vamos a usar para esta parte del mini curso</span></span>
<span id="cb32-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb32-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb32-3" aria-hidden="true" tabindex="-1"></a><span class="do">## Infraestructura</span></span>
<span id="cb32-4"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb32-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;SummarizedExperiment&quot;</span>)</span></code></pre></div>
<pre><code>## Loading required package: MatrixGenerics</code></pre>
<pre><code>## Loading required package: matrixStats</code></pre>
<pre><code>## 
## Attaching package: &#39;matrixStats&#39;</code></pre>
<pre><code>## The following object is masked from &#39;package:dplyr&#39;:
## 
##     count</code></pre>
<pre><code>## 
## Attaching package: &#39;MatrixGenerics&#39;</code></pre>
<pre><code>## The following objects are masked from &#39;package:matrixStats&#39;:
## 
##     colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse, colCounts, colCummaxs, colCummins,
##     colCumprods, colCumsums, colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs, colMads, colMaxs,
##     colMeans2, colMedians, colMins, colOrderStats, colProds, colQuantiles, colRanges, colRanks, colSdDiffs,
##     colSds, colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads, colWeightedMeans,
##     colWeightedMedians, colWeightedSds, colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
##     rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods, rowCumsums, rowDiffs, rowIQRDiffs,
##     rowIQRs, rowLogSumExps, rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins, rowOrderStats,
##     rowProds, rowQuantiles, rowRanges, rowRanks, rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs,
##     rowVars, rowWeightedMads, rowWeightedMeans, rowWeightedMedians, rowWeightedSds, rowWeightedVars</code></pre>
<pre><code>## Loading required package: GenomicRanges</code></pre>
<pre><code>## Loading required package: stats4</code></pre>
<pre><code>## Loading required package: BiocGenerics</code></pre>
<pre><code>## Loading required package: parallel</code></pre>
<pre><code>## 
## Attaching package: &#39;BiocGenerics&#39;</code></pre>
<pre><code>## The following objects are masked from &#39;package:parallel&#39;:
## 
##     clusterApply, clusterApplyLB, clusterCall, clusterEvalQ, clusterExport, clusterMap, parApply,
##     parCapply, parLapply, parLapplyLB, parRapply, parSapply, parSapplyLB</code></pre>
<pre><code>## The following objects are masked from &#39;package:dplyr&#39;:
## 
##     combine, intersect, setdiff, union</code></pre>
<pre><code>## The following objects are masked from &#39;package:stats&#39;:
## 
##     IQR, mad, sd, var, xtabs</code></pre>
<pre><code>## The following objects are masked from &#39;package:base&#39;:
## 
##     anyDuplicated, append, as.data.frame, basename, cbind, colnames, dirname, do.call, duplicated, eval,
##     evalq, Filter, Find, get, grep, grepl, intersect, is.unsorted, lapply, Map, mapply, match, mget, order,
##     paste, pmax, pmax.int, pmin, pmin.int, Position, rank, rbind, Reduce, rownames, sapply, setdiff, sort,
##     table, tapply, union, unique, unsplit, which.max, which.min</code></pre>
<pre><code>## Loading required package: S4Vectors</code></pre>
<pre><code>## 
## Attaching package: &#39;S4Vectors&#39;</code></pre>
<pre><code>## The following object is masked from &#39;package:plotly&#39;:
## 
##     rename</code></pre>
<pre><code>## The following objects are masked from &#39;package:dplyr&#39;:
## 
##     first, rename</code></pre>
<pre><code>## The following object is masked from &#39;package:tidyr&#39;:
## 
##     expand</code></pre>
<pre><code>## The following object is masked from &#39;package:base&#39;:
## 
##     expand.grid</code></pre>
<pre><code>## Loading required package: IRanges</code></pre>
<pre><code>## 
## Attaching package: &#39;IRanges&#39;</code></pre>
<pre><code>## The following object is masked from &#39;package:plotly&#39;:
## 
##     slice</code></pre>
<pre><code>## The following objects are masked from &#39;package:dplyr&#39;:
## 
##     collapse, desc, slice</code></pre>
<pre><code>## The following object is masked from &#39;package:purrr&#39;:
## 
##     reduce</code></pre>
<pre><code>## Loading required package: GenomeInfoDb</code></pre>
<pre><code>## Loading required package: Biobase</code></pre>
<pre><code>## Welcome to Bioconductor
## 
##     Vignettes contain introductory material; view with &#39;browseVignettes()&#39;. To cite Bioconductor, see
##     &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.</code></pre>
<pre><code>## 
## Attaching package: &#39;Biobase&#39;</code></pre>
<pre><code>## The following object is masked from &#39;package:MatrixGenerics&#39;:
## 
##     rowMedians</code></pre>
<pre><code>## The following objects are masked from &#39;package:matrixStats&#39;:
## 
##     anyMissing, rowMedians</code></pre>
<div class="sourceCode" id="cb65"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb65-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;SingleCellExperiment&quot;</span>)</span>
<span id="cb65-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-2" aria-hidden="true" tabindex="-1"></a><span class="do">## library(&quot;SpatialExperiment&quot;) ## A partir de Bioconductor 3.13</span></span>
<span id="cb65-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-3" aria-hidden="true" tabindex="-1"></a><span class="do">## que estar√° disponible a finales de mayo 2021</span></span>
<span id="cb65-4"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb65-5"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-5" aria-hidden="true" tabindex="-1"></a><span class="do">## Para descargar y procesar datos</span></span>
<span id="cb65-6"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;scRNAseq&quot;</span>)</span>
<span id="cb65-7"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;scater&quot;</span>)</span>
<span id="cb65-8"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;Rtsne&quot;</span>)</span>
<span id="cb65-9"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-9" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;recount3&quot;</span>)</span>
<span id="cb65-10"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb65-11"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-11" aria-hidden="true" tabindex="-1"></a><span class="do">## Para visualizar los datos</span></span>
<span id="cb65-12"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb65-12" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;iSEE&quot;</span>)</span></code></pre></div>
<div id="infraestructura-para-datos-tr√°nscriptomicos" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Infraestructura para datos tr√°nscriptomicos</h2>
<ul>
<li><a href="https://bioconductor.org/packages/SummarizedExperiment">SummarizedExperiment</a></li>
</ul>
<p><img src="https://www.researchgate.net/profile/Pascal-Martin-6/publication/274256021/figure/fig3/AS:294713816043527@1447276657416/Structure-of-summarizedExperiment-objects.png" width="600px" /></p>
<ul>
<li><a href="https://bioconductor.org/packages/SingleCellExperiment">SingleCellExperiment</a></li>
</ul>
<p><img src="https://raw.githubusercontent.com/Bioconductor/OSCABase/images/images/SingleCellExperiment.png" width="600px" /></p>
<ul>
<li>Revisemos las diapositivas 31 a 43 de la siguiente presentaci√≥n</li>
</ul>
<blockquote class="twitter-tweet">
<p lang="en" dir="ltr">
Thanks to Pete <a href="https://twitter.com/PeteHaitch?ref_src=twsrc%5Etfw"><span class="citation">@PeteHaitch</span></a> &amp; <a href="https://twitter.com/hashtag/OSCA?src=hash&amp;ref_src=twsrc%5Etfw">#OSCA</a> ‚ù§Ô∏è I'm now ready for my Human Cell Atlas - Latin America presentation on ‚ÄúGetting started with <a href="https://twitter.com/hashtag/scRNA?src=hash&amp;ref_src=twsrc%5Etfw">#scRNA</a>-seq analyses with <a href="https://twitter.com/Bioconductor?ref_src=twsrc%5Etfw"><span class="citation">@Bioconductor</span></a>‚Äù üéâ<br><br>üìî <a href="https://t.co/nEWZcnadyh">https://t.co/nEWZcnadyh</a><br>üíª <a href="https://t.co/7pXDTQRpF0">https://t.co/7pXDTQRpF0</a><a href="https://twitter.com/hashtag/rstats?src=hash&amp;ref_src=twsrc%5Etfw">#rstats</a> <a href="https://twitter.com/CDSBMexico?ref_src=twsrc%5Etfw"><span class="citation">@CDSBMexico</span></a> <a href="https://twitter.com/humancellatlas?ref_src=twsrc%5Etfw"><span class="citation">@humancellatlas</span></a> <a href="https://twitter.com/hashtag/LatAm?src=hash&amp;ref_src=twsrc%5Etfw">#LatAm</a> <a href="https://t.co/Otg8rdSlYW">pic.twitter.com/Otg8rdSlYW</a>
</p>
‚Äî üá≤üáΩ Leonardo Collado-Torres (<span class="citation">@lcolladotor</span>) <a href="https://twitter.com/lcolladotor/status/1388934387090305025?ref_src=twsrc%5Etfw">May 2, 2021</a>
</blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
<ul>
<li>Incluso va m√°s all√° con <a href="https://bioconductor.org/packages/SpatialExperiment">SpatialExperiment</a></li>
</ul>
<blockquote class="twitter-tweet">
<p lang="en" dir="ltr">
Are you working with spatial transcriptomics data such as Visium from <a href="https://twitter.com/10xGenomics?ref_src=twsrc%5Etfw"><span class="citation">@10xGenomics</span></a>? Then you'll be interested in <a href="https://twitter.com/hashtag/SpatialExperiment?src=hash&amp;ref_src=twsrc%5Etfw">#SpatialExperiment</a> üì¶ led by <a href="https://twitter.com/drighelli?ref_src=twsrc%5Etfw"><span class="citation">@drighelli</span></a> <a href="https://twitter.com/lmwebr?ref_src=twsrc%5Etfw"><span class="citation">@lmwebr</span></a> <a href="https://twitter.com/CrowellHL?ref_src=twsrc%5Etfw"><span class="citation">@CrowellHL</span></a> with contributions by <a href="https://twitter.com/PardoBree?ref_src=twsrc%5Etfw"><span class="citation">@PardoBree</span></a> <a href="https://twitter.com/shazanfar?ref_src=twsrc%5Etfw"><span class="citation">@shazanfar</span></a> A Lun <a href="https://twitter.com/stephaniehicks?ref_src=twsrc%5Etfw"><span class="citation">@stephaniehicks</span></a> <a href="https://twitter.com/drisso1893?ref_src=twsrc%5Etfw"><span class="citation">@drisso1893</span></a> üåü<br><br>üìú <a href="https://t.co/r36qlakRJe">https://t.co/r36qlakRJe</a> <a href="https://t.co/cWIiwLFitV">pic.twitter.com/cWIiwLFitV</a>
</p>
‚Äî üá≤üáΩ Leonardo Collado-Torres (<span class="citation">@lcolladotor</span>) <a href="https://twitter.com/lcolladotor/status/1355208674856329218?ref_src=twsrc%5Etfw">January 29, 2021</a>
</blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
<ul>
<li>Usamos <code>SpatialExperiment</code> para hacer <a href="https://bioconductorl.org/packages/spatialLIBD">spatialLIBD</a>. En este proyecto particip√≥ Brenda Pardo, que es una estudiante de tercer a√±o de la LCG-EJ-UNAM en el LIIGH-UNAM.</li>
</ul>
<blockquote class="twitter-tweet">
<p lang="en" dir="ltr">
Our paper describing our package <a href="https://twitter.com/hashtag/spatialLIBD?src=hash&amp;ref_src=twsrc%5Etfw">#spatialLIBD</a> is finally out! üéâüéâüéâ<br><br>spatialLIBD is an <a href="https://twitter.com/hashtag/rstats?src=hash&amp;ref_src=twsrc%5Etfw">#rstats</a> / <a href="https://twitter.com/Bioconductor?ref_src=twsrc%5Etfw"><span class="citation">@Bioconductor</span></a> package to visualize spatial transcriptomics data.<br>‚Å∞<br>This is especially exciting for me as it is my first paper as a first author ü¶ë.<a href="https://t.co/COW013x4GA">https://t.co/COW013x4GA</a><br><br>1/9 <a href="https://t.co/xevIUg3IsA">pic.twitter.com/xevIUg3IsA</a>
</p>
‚Äî Brenda Pardo (<span class="citation">@PardoBree</span>) <a href="https://twitter.com/PardoBree/status/1388253938391175173?ref_src=twsrc%5Etfw">April 30, 2021</a>
</blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
</div>
<div id="datos-p√∫blicos" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Datos p√∫blicos</h2>
<ul>
<li><code>scRNAseq</code></li>
</ul>
<p><a href="https://bioconductor.org/packages/scRNAseq">scRNAseq</a> es un paquete de Bioconductor que cre√≥ Aaron Lun para poder tener varios datos de ejemplo de scRNA-seq (secuenciaci√≥n masiva de ARN de c√©lulas √∫nicas) para poder demostrar sus m√©todos estad√≠sticos. Ese paquete lo usan en <code>iSEE</code> para obtener datos de ejemplo y tambi√©n lo usan mucho en <em>Orchestrating Single Cell Analysis (OSCA) with Bioconductor</em> <a href="http://bioconductor.org/books/release/OSCA/" class="uri">http://bioconductor.org/books/release/OSCA/</a>.</p>
<p>A continuaci√≥n vamos a descargar unos datos de scRNA-seq que usan en los ejemplos de <code>iSEE</code> en <a href="http://bioconductor.org/packages/release/bioc/vignettes/iSEE/inst/doc/basic.html#2_Setting_up_the_data" class="uri">http://bioconductor.org/packages/release/bioc/vignettes/iSEE/inst/doc/basic.html#2_Setting_up_the_data</a>. <code>scRNAseq</code> usa otro paquete de Bioconductor llamado <code>ExperimentHub</code> para descargar los archivos y organizarlos en un lugar de tal forma que si corremos el c√≥digo otra vez en la misma computadora, no tendremos que volver a descargar los archivos. Es decir, crea un cache local.</p>
<div class="sourceCode" id="cb66"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb66-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb66-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Descarguemos datos de ejemplo de scRNA-seq del proyecto de</span></span>
<span id="cb66-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb66-2" aria-hidden="true" tabindex="-1"></a><span class="do">## Tasic et al 2016</span></span>
<span id="cb66-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb66-3" aria-hidden="true" tabindex="-1"></a>sce <span class="ot">&lt;-</span> <span class="fu">ReprocessedAllenData</span>(<span class="at">assays =</span> <span class="st">&quot;tophat_counts&quot;</span>)</span>
<span id="cb66-4"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb66-4" aria-hidden="true" tabindex="-1"></a>sce</span></code></pre></div>
<pre><code>## class: SingleCellExperiment 
## dim: 20816 379 
## metadata(2): SuppInfo which_qc
## assays(1): tophat_counts
## rownames(20816): 0610007P14Rik 0610009B22Rik ... Zzef1 Zzz3
## rowData names(0):
## colnames(379): SRR2140028 SRR2140022 ... SRR2139341 SRR2139336
## colData names(22): NREADS NALIGNED ... Animal.ID passes_qc_checks_s
## reducedDimNames(0):
## altExpNames(1): ERCC</code></pre>
<p>Los datos que descargamos son datos digamos que crudos, as√≠ que necesitamos correr un par de an√°lisis usando <code>scater</code> para que sea m√°s interesante el ejemplo. No es el objetivo de este mini curso explicar esos pasos, aunque a grandes rasgos, nos van a ayudar a que podamos comparar las diferentes c√©lulas. Tambi√©n nos ayudar√° a visualizar las c√©lulas.</p>
<div class="sourceCode" id="cb68"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb68-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Normalizamos los datos</span></span>
<span id="cb68-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-2" aria-hidden="true" tabindex="-1"></a>sce <span class="ot">&lt;-</span> <span class="fu">logNormCounts</span>(sce, <span class="at">exprs_values =</span> <span class="st">&quot;tophat_counts&quot;</span>)</span>
<span id="cb68-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb68-4"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-4" aria-hidden="true" tabindex="-1"></a><span class="do">## Obtenemos dimensiones reducidas usando primero un an√°lisis de componentes</span></span>
<span id="cb68-5"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-5" aria-hidden="true" tabindex="-1"></a><span class="do">## principales y luego con tSNE</span></span>
<span id="cb68-6"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-6" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">1000</span>)</span>
<span id="cb68-7"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-7" aria-hidden="true" tabindex="-1"></a>sce <span class="ot">&lt;-</span> <span class="fu">runPCA</span>(sce)</span>
<span id="cb68-8"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-8" aria-hidden="true" tabindex="-1"></a>sce <span class="ot">&lt;-</span> <span class="fu">runTSNE</span>(sce)</span>
<span id="cb68-9"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb68-10"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-10" aria-hidden="true" tabindex="-1"></a><span class="do">## Ahora tenemos datos de PCA y TSNE en nuestro objeto de R</span></span>
<span id="cb68-11"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb68-11" aria-hidden="true" tabindex="-1"></a><span class="fu">reducedDimNames</span>(sce)</span></code></pre></div>
<pre><code>## [1] &quot;PCA&quot;  &quot;TSNE&quot;</code></pre>
<div class="sourceCode" id="cb70"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb70-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb70-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Podemos ahora calcular unas variables a lo largo de nuestros genes</span></span>
<span id="cb70-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb70-2" aria-hidden="true" tabindex="-1"></a><span class="do">## tal como la expresi√≥n media del gene en las c√©lulas</span></span>
<span id="cb70-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb70-3" aria-hidden="true" tabindex="-1"></a><span class="do">## y la varianza</span></span>
<span id="cb70-4"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb70-4" aria-hidden="true" tabindex="-1"></a><span class="fu">rowData</span>(sce)<span class="sc">$</span>mean_log <span class="ot">&lt;-</span> <span class="fu">rowMeans</span>(<span class="fu">logcounts</span>(sce))</span>
<span id="cb70-5"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb70-5" aria-hidden="true" tabindex="-1"></a><span class="fu">rowData</span>(sce)<span class="sc">$</span>var_log <span class="ot">&lt;-</span> <span class="fu">apply</span>(<span class="fu">logcounts</span>(sce), <span class="dv">1</span>, var)</span>
<span id="cb70-6"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb70-6" aria-hidden="true" tabindex="-1"></a><span class="fu">rowData</span>(sce)</span></code></pre></div>
<pre><code>## DataFrame with 20816 rows and 2 columns
##                mean_log   var_log
##               &lt;numeric&gt; &lt;numeric&gt;
## 0610007P14Rik 7.9369535   9.37109
## 0610009B22Rik 6.7572139  10.80402
## 0610009L18Rik 0.0881306   0.40976
## 0610009O20Rik 3.6084644  14.97973
## 0610010F05Rik 6.5254121  13.45587
## ...                 ...       ...
## Zyg11a        0.0854769  0.109352
## Zyg11b        8.9813448  3.988757
## Zyx           2.4291810 11.860900
## Zzef1         2.9058253 11.223440
## Zzz3          4.4118629 15.179005</code></pre>
<p>Ahora nuestro objeto <code>sce</code> es m√°s complejo y tiene m√°s informaci√≥n que podremos visualizar con <code>iSEE</code>.</p>
<ul>
<li><code>recount3</code></li>
</ul>
<p>Para que tengan m√°s datos de ejemplo, usaremos datos del proyecto de <code>recount3</code> v√≠a el cual pueden descargar informaci√≥n de m√°s de 700,000 muestras de RNA-seq de rat√≥n o humano tal como describimos en <a href="http://rna.recount.bio/" class="uri">http://rna.recount.bio/</a>. La idea es que pueden obtener objetos de R del tipo <code>RangedSummarizedExperiment</code> que tambi√©n podemos visualizar con <code>iSEE</code>.</p>
<p><a href="https://f1000research.com/articles/6-1558"><img src="https://f1000researchdata.s3.amazonaws.com/manuscripts/13233/8a00ef7e-4c4b-4d53-bd4f-2a28a23ec216_figure2.gif" /></a></p>
<div class="sourceCode" id="cb72"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb72-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb72-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Revisemos todos los proyectos con datos de humano en recount3</span></span>
<span id="cb72-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb72-2" aria-hidden="true" tabindex="-1"></a>human_projects <span class="ot">&lt;-</span> <span class="fu">available_projects</span>()</span>
<span id="cb72-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb72-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-4"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb72-4" aria-hidden="true" tabindex="-1"></a><span class="do">## Veamos cuantos proyectos hay</span></span>
<span id="cb72-5"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb72-5" aria-hidden="true" tabindex="-1"></a><span class="fu">nrow</span>(human_projects)</span></code></pre></div>
<pre><code>## [1] 8742</code></pre>
<div class="sourceCode" id="cb74"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb74-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Encuentra tu proyecto de inter√©s. Aqu√≠ usaremos</span></span>
<span id="cb74-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-2" aria-hidden="true" tabindex="-1"></a><span class="do">## SRP009615 de ejemplo</span></span>
<span id="cb74-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-3" aria-hidden="true" tabindex="-1"></a>proj_info <span class="ot">&lt;-</span> <span class="fu">subset</span>(</span>
<span id="cb74-4"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-4" aria-hidden="true" tabindex="-1"></a>    human_projects,</span>
<span id="cb74-5"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-5" aria-hidden="true" tabindex="-1"></a>    project <span class="sc">==</span> <span class="st">&quot;SRP009615&quot;</span> <span class="sc">&amp;</span> project_type <span class="sc">==</span> <span class="st">&quot;data_sources&quot;</span></span>
<span id="cb74-6"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-6" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb74-7"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-7" aria-hidden="true" tabindex="-1"></a><span class="do">## Crea un objetio de tipo RangedSummarizedExperiment (RSE)</span></span>
<span id="cb74-8"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-8" aria-hidden="true" tabindex="-1"></a><span class="do">## con la informaci√≥n a nivel de genes</span></span>
<span id="cb74-9"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-9" aria-hidden="true" tabindex="-1"></a>rse_gene_SRP009615 <span class="ot">&lt;-</span> <span class="fu">create_rse</span>(proj_info)</span>
<span id="cb74-10"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-10" aria-hidden="true" tabindex="-1"></a><span class="do">## Explora el objeto RSE</span></span>
<span id="cb74-11"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb74-11" aria-hidden="true" tabindex="-1"></a>rse_gene_SRP009615</span></code></pre></div>
<pre><code>## class: RangedSummarizedExperiment 
## dim: 63856 12 
## metadata(8): time_created recount3_version ... annotation recount3_url
## assays(1): raw_counts
## rownames(63856): ENSG00000278704.1 ENSG00000277400.1 ... ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y
## rowData names(10): source type ... havana_gene tag
## colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
## colData names(175): rail_id external_id ... recount_pred.curated.cell_line BigWigURL</code></pre>
<p>Los objetos de <code>recount3</code> tienen ciertas propiedades que aqu√≠ no explicaremos. Pero bueno, b√°sicamente los siguientes comandos los querr√°n usar con los diferentes datos que descarguen de <code>recount3</code>.</p>
<div class="sourceCode" id="cb76"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb76-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb76-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Convirtamos las cuentas por nucleotido a cuentas por lectura</span></span>
<span id="cb76-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb76-2" aria-hidden="true" tabindex="-1"></a><span class="do">## usando compute_read_counts().</span></span>
<span id="cb76-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb76-3" aria-hidden="true" tabindex="-1"></a><span class="do">## Para otras transformaciones como RPKM y TPM, revisa transform_counts().</span></span>
<span id="cb76-4"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb76-4" aria-hidden="true" tabindex="-1"></a><span class="fu">assay</span>(rse_gene_SRP009615, <span class="st">&quot;counts&quot;</span>) <span class="ot">&lt;-</span> <span class="fu">compute_read_counts</span>(rse_gene_SRP009615)</span></code></pre></div>
<div class="sourceCode" id="cb77"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb77-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb77-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Para este estudio en espec√≠fico, hagamos m√°s f√°cil de usar la</span></span>
<span id="cb77-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb77-2" aria-hidden="true" tabindex="-1"></a><span class="do">## informaci√≥n del experimento</span></span>
<span id="cb77-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb77-3" aria-hidden="true" tabindex="-1"></a>rse_gene_SRP009615 <span class="ot">&lt;-</span> <span class="fu">expand_sra_attributes</span>(rse_gene_SRP009615)</span>
<span id="cb77-4"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb77-4" aria-hidden="true" tabindex="-1"></a><span class="fu">colData</span>(rse_gene_SRP009615)[</span>
<span id="cb77-5"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb77-5" aria-hidden="true" tabindex="-1"></a>    ,</span>
<span id="cb77-6"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb77-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">grepl</span>(<span class="st">&quot;^sra_attribute&quot;</span>, <span class="fu">colnames</span>(<span class="fu">colData</span>(rse_gene_SRP009615)))</span>
<span id="cb77-7"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb77-7" aria-hidden="true" tabindex="-1"></a>]</span></code></pre></div>
<pre><code>## DataFrame with 12 rows and 4 columns
##           sra_attribute.cells sra_attribute.shRNA_expression sra_attribute.source_name sra_attribute.treatment
##                   &lt;character&gt;                    &lt;character&gt;               &lt;character&gt;             &lt;character&gt;
## SRR387777                K562                             no                    SL2933               Puromycin
## SRR387778                K562             yes, targeting SRF                    SL2934  Puromycin, doxycycline
## SRR387779                K562                             no                    SL5265               Puromycin
## SRR387780                K562              yes targeting SRF                    SL3141  Puromycin, doxycycline
## SRR389079                K562            no shRNA expression                    SL6485               Puromycin
## ...                       ...                            ...                       ...                     ...
## SRR389082                K562         expressing shRNA tar..                    SL2592  Puromycin, doxycycline
## SRR389083                K562            no shRNA expression                    SL4337               Puromycin
## SRR389084                K562         expressing shRNA tar..                    SL4326  Puromycin, doxycycline
## SRR389077                K562            no shRNA expression                    SL1584               Puromycin
## SRR389078                K562         expressing shRNA tar..                    SL1583  Puromycin, doxycycline</code></pre>
</div>
<div id="explorando-con-isee" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Explorando con iSEE</h2>
<div id="sce" class="section level3" number="2.3.1">
<h3><span class="header-section-number">2.3.1</span> sce</h3>
<div class="sourceCode" id="cb79"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb79-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb79-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="fu">interactive</span>()) {</span>
<span id="cb79-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb79-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">iSEE</span>(sce, <span class="at">appTitle =</span> <span class="st">&quot;Aprendiendo a usar iSEE&quot;</span>)</span>
<span id="cb79-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb79-3" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<ul>
<li>Exploremos varias variables (column data) en los PCA y TSNE. ¬øQu√© variable separa los grupos de muestras?</li>
<li>¬øQu√© muestra tiene la mayor cantidad de n√∫mero de lecturas (NREADS)?</li>
<li>Aprendamos a hacer _facet_s en el panel que muestra informaci√≥n de los genes (feature assay plot 1) y a ligar ese panel a la tabla con informaci√≥n de los genes</li>
<li>Comparemos el n√∫mero de lecturas (NREADS) usando _facet_s por disecci√≥n. ¬øAlguna nos llama la atenci√≥n? Podemos usar la columna de ‚Äúpasses qc checks‚Äù.</li>
<li>Hagamos una gr√°fica de la media (eje X) vs la varianza (eje Y) de expresi√≥n de los genes. ¬øVemos alg√∫n patr√≥n?</li>
</ul>
</div>
<div id="ejercicio" class="section level3" number="2.3.2">
<h3><span class="header-section-number">2.3.2</span> Ejercicio</h3>
<ul>
<li>Ejercicio: <a href="https://lcolladotor.github.io/rnaseq_LCG-UNAM_2021/datos-de-rna-seq-a-trav%C3%A9s-de-recount3.html#ejercicio-1" class="uri">https://lcolladotor.github.io/rnaseq_LCG-UNAM_2021/datos-de-rna-seq-a-trav%C3%A9s-de-recount3.html#ejercicio-1</a></li>
</ul>
<div class="sourceCode" id="cb80"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb80-1"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb80-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="fu">interactive</span>()) {</span>
<span id="cb80-2"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb80-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">iSEE</span>(rse_gene_SRP009615, <span class="at">appTitle =</span> <span class="st">&quot;iSEE con datos de recount3&quot;</span>)</span>
<span id="cb80-3"><a href="exploraci√≥n-interactiva-de-datos-transcript√≥micos-con-isee.html#cb80-3" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
<div id="datos-reales" class="section level3" number="2.3.3">
<h3><span class="header-section-number">2.3.3</span> Datos reales</h3>
<p>Datos de <a href="https://www.biorxiv.org/content/10.1101/2020.10.07.329839v1.full" class="uri">https://www.biorxiv.org/content/10.1101/2020.10.07.329839v1.full</a> que pueden ver en pag√≠nas web creadas con <code>iSEE</code> a trav√©s de <a href="https://github.com/LieberInstitute/10xPilot_snRNAseq-human#explore-the-data-interactively" class="uri">https://github.com/LieberInstitute/10xPilot_snRNAseq-human#explore-the-data-interactively</a>. Para lograr esto subimos el resultado de <code>iSEE</code> a <a href="https://shinyapps.io" class="uri">https://shinyapps.io</a>.</p>
</div>
</div>
<div id="m√°s-material" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> M√°s material</h2>
<p>Si les llam√≥ la atenci√≥n <code>iSEE</code> y <code>recount3</code>, v√≠a <a href="https://lcolladotor.github.io/rnaseq_LCG-UNAM_2021" class="uri">https://lcolladotor.github.io/rnaseq_LCG-UNAM_2021</a> pueden aprender m√°s sobre estos paquetes.</p>
<p><a href="https://comunidadbioinfo.github.io/es/post/cs_and_s_event_fund_award/#.YJH-wbVKj8A"><img src="https://comunidadbioinfo.github.io/post/2021-01-27-cs_and_s_event_fund_award/spanish_cs_and_s_award.jpeg" width="400px" align="center"/></a></p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="exploraci√≥n-interactiva-de-datos-con-plotly.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
